ENIGMAsystem/SHELL/Graphics_Systems/General/GScolor_macros.h

```diff
@@ -0,0 +1,20 @@
+#ifndef ENIGMA_GSCOLOR_MACROS_H
+#define ENIGMA_GSCOLOR_MACROS_H
+

```

Comment:
add a comment to what each of these groups do differently. especially the last one

Reply:
Did you move these here from somewhere? Let's avoid identifiers containing `__`/starting with `_ABC` going forward.